### ISSUE TITLE
Colour accessibility

### ISSUE THREAD

**Issue by kate-miller** [cesensw/school-finder#3]
Hi guys,

It's been raised that the icons for the School Finder are not accessible against the map. I've done some testing, and the following colours appear to pass AA accessibility for large text:

- ssp: #518A64 (green)
- primary: #E03838 (red-ish)
- secondary: #B940C9 (purple)
- central: #B940C9 (purple)
- infants: #2A85B7 (blue)
- environmental: #DC1461 (pink)
- other: #DC1461 (pink)

I changed environmental and other to be pink because they were the same blue colour as infants. 

What do you think of these colour choices?
